lib/src/realm_class.dart

```
@@ -62,6 +63,11 @@ class Realm {
     }
   }
 
+  static void deleteRealm(Configuration config) {
+    // TODO: To discuss: Do we need to return this bool result from realmCore?
```

Why not? This is IO, the User needs to know if operation succeeded or not

---

this should throw a `RealmException` if it fails, imo

---

That is fine with me as well. As long as it cannot fail silently.

---

It won't fail silently, but it may succeed even if an exception is called, which is why there's a return value.

---

I mean on the dart side.. Or maybe I misunderstand?

---

After a discussion we decided to leave the exception to be thrown by `invokeGetBool`. Return type will be void.